### PR TITLE
Add vSphere remote worker/nutanix extend cron job

### PR DESCRIPTION
**Pull request by sgaoshang**

### CODE REVIEW

**Comment by sgaoshang** [comment]
@jianlinliu pls help to review, thanks.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -2857,6 +2857,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones
+- as: vsphere-ipi-ipi-remote-worker-p2-f13
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2860]: 1. `f13`? not `f14`?
2. The ci jobs in the yaml file are placed in alphabetical order, so pls correct it.

**Comment by jianlinliu** [comment]
@sgaoshang do you mind to add the ci coverage for `nutanix-ipi-boot-categories-project` together in this PR?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 2]
```diff
@@ -2680,6 +2680,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-private-fips
+- as: nutanix-ipi-boot-categories-project-p2-f14
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2683]: How about set it to `f7` and update `nutanix-ipi-p2-f3` to `nutanix-ipi-p2-f7`

**Comment by jianlinliu** [comment]
@sgaoshang pls also copy the ci jobs to 4.14, too.

**Comment by jianlinliu** [comment]
BTW, suggest to update the PR's title togther.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 3]
```diff
@@ -2680,9 +2680,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-private-fips
-- as: nutanix-ipi-p2-f3
+- as: nutanix-ipi-boot-categories-project-p2-f7
   cluster: build01
-  cron: 4 19 3,7,10,13,16,19,22,25,28,31 * *
+  cron: 55 11 1,8,15,23 * *
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2685]: Hmm, I tried to generate the cron locally, but seem like get different value with yours.
```
$ generate-cron-entry.sh nutanix-ipi-boot-categories-project-p2-f7 openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml
cron: 15 14 6,14,22,29 * *
```

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**